src/javascript/JContent/ContentRoute/ToolBar/Toolbar.test.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {shallow} from '@jahia/test-framework';
+import ToolBar from './ToolBar';
+import {useSelector} from 'react-redux';
+
+jest.mock('react-redux', () => ({
+    useDispatch: jest.fn(),
+    useSelector: jest.fn()
+}));
+
+jest.mock('~/JContent/JContent.redux', () => ({
+    cmClearSelection: jest.fn()
+}));
+
+jest.mock('connected-react-router', () => jest.fn(() => {}));
+
+describe('Toolbar', () => {
+    it('should not render multiple selection actions', () => {
+        useSelector.mockImplementation(() => ({
+            selection: []
+        }));
+
+        const toolbar = shallow(<ToolBar/>);
+        expect(toolbar.find('Typography').length).toEqual(0);
```

Comment:
Could with have dedicated tests for the "selection-infos" instead? It's not clear what all those Typography related tests are about while just looking at the tests here.

You could have one test that check that the message is not displayed when there is not selection, and another one testing that it is displayed if there is any selection.

Reply:
sure